Automattic/sensei
# Add logging for Settings page view

Closes #2644 

Note that the `view` property values differ from what the issue specified in order to match the tab slugs used by the settings page and JS code.

This PR also adds the ability to log events in WP Admin from JS code using the `sensei_log_event` function.

## Testing Instructions

- Visit the Settings page. Ensure the `sensei_settings_view` event was logged with a `view` of `default-settings`.
- Visit each of the tabs on the Settings page. Ensure that an event is fired for each tab, with the correct `view` value.
